py/desimodel/install.py

```diff
@@ -32,36 +32,69 @@ def assert_svn_exists():
     except CalledProcessError as e:
         raise AssertionError("The svn command ({0}) on this system does not work. Output is: '{1}'.".format(e.cmd, e.output))
 
+def get_svn_version(desimodel_version=None):
+    """
+    Return which svn version should be checked out for given desimodel_version
+
+    Parameters
+    ----------
+    desimodel_version : :class:`str`, optional
+        The version X.Y.Z to download, trunk, or something of the
+        form branches/... Defaults to package version if x.y.z tagged,
+        otherwise trunk.
 
-def svn_export(desimodel_version=None):
+    Returns
+    -------
+    :class:`str`
+        svn version string
+    """
+
+    if desimodel_version is None:
```

Comment:
Many packages make this check much more simply: `if 'dev' in this_version:  # ... not a tag`.

Reply:
That's what I originally implemented, and that works fine for our standard version naming procedures, but I felt that the more restrictive explicit check for X.Y.Z or X.Y was more robust.  Admittedly, this still has the fragility of assuming that developers properly followed procedures (when making a code tag, also make a matching data tag), so maybe it doesn't matter and I wouldn't bother changing other code checking for 'dev' unless we actually start tripping on that.